WalletWasabi.Gui/CrashReport/CrashReporter.cs

```diff
@@ -31,8 +31,8 @@ public void TryInvokeCrashReport()
 
 				var args = $"crashreport -attempt=\"{Attempts + 1}\" -exception=\"{Base64ExceptionString}\"";
 
-				ProcessBridge processBridge = new ProcessBridge(Process.GetCurrentProcess().MainModule.FileName);
-				processBridge.Start(args, false);
+				ProcessStartInfo startInfo = ProcessStartInfoFactory.Make(Process.GetCurrentProcess().MainModule.FileName, args);
```

Comment:
This looks like a crazy pattern.

Reply:
I hate that word `pattern`. It's misused ad nauseam. 

`ProcessStartInfoFactory.Make()` creates `ProcessStartInfo` the way Wasabi Wallet typically does. I'm not exactly sure why those exact parameters inside were chosen but it's the way it is. 

The point here is that you can do with thath `ProcessStartInfo` whatever you want. When `ProcessStartInfo` is buried inside `ProcessBridge`, it's very hard to pass any parameters or modify its behavior. So that's the motivation behind this change.

Reply:
Quoting you:

> Provide nice API for Wasabi developers so that it's easy to use

You aren't really achieving your goal, quite the opposite, you are making it even harder to use.